Makefile

```
@@ -157,9 +157,10 @@ _build.operator:
 build: generate
 	$(MAKE) build.operator
 
+GOLANGCI_LINT_CONFIG = $(PROJECT_DIR)/.golangci.yaml
```

I believe we want `?=` to allow overrides, right?

```suggestion
GOLANGCI_LINT_CONFIG ?= $(PROJECT_DIR)/.golangci.yaml
```